pageserver/src/walreceiver/connection_manager.rs

```diff
@@ -558,41 +631,31 @@ impl WalreceiverState {
         &self,
         node_to_omit: Option<NodeId>,
     ) -> Option<(NodeId, &SkTimelineInfo, String)> {
-        let all_candidates = self
-            .applicable_connection_candidates()
+        self.applicable_connection_candidates()
```

Comment:
NIT: looks line we can inline the method now, it's name does not add more sense and the method is used once only.